contribution/plan.md

```diff
@@ -0,0 +1,11 @@
+Project Contributions
```

Comment:
remove this file or put contents in `ks3651.md`

Reply:
I removed the "contributions" folder and left only the "projects" folder which has a javascript folder for the contribution and a python folder for the project.

I attached each link to the ks3651.md as the professor's md file.